core/src/cats/cats.h

```diff
@@ -497,7 +498,7 @@ class BareosDb : public BareosDbQueryEnum {
   uint32_t ref_count_ = 0;                /**< Reference count */
   bool connected_ = false;                /**< Connection made to db */
   bool have_batch_insert_ = false;        /**< Have batch insert support ? */
-  bool try_reconnect_ = false;   /**< Try reconnecting DB connection ? */
+  bool try_reconnect_ = true;    /**< Try reconnecting DB connection ? */
```

Comment:
as every user of BareosDb is required to pass in `try_reconnect` when constructing, this won't make a difference